tools/cli/flags.go

```diff
@@ -406,6 +406,43 @@ func getFlagsForListAll() []cli.Flag {
 	}
 }
 
+func getFlagsForScan() []cli.Flag {
+	return []cli.Flag{
+		cli.IntFlag{
+			Name:  FlagPageSizeWithAlias,
```

Comment:
you can set the default value here so that the --help will show the default value